README.md

```diff
@@ -286,6 +286,32 @@ expect('hello world').toExclude('goodbye')
 expect('hello world').toNotContain('goodbye')
 ```
 
+### toIncludeKeys
+> `expect(object).toIncludeKeys(keys, [hasKey], [message])`<br>
```

Comment:
I believe `toInclude` and `toExclude` only deal with own properties - I think we should be consistent here. In addition, "own" should be the default, I'd think.

I'd also want a separate method to deal with inherited properties, rather than a boolean switch.